src/components/security/Main.tsx

```diff
@@ -169,7 +169,7 @@ const MainSection = () => {
             </a>
           </UsdWorthDate>
           <SButtonLink
-            url="https://explore.duneanalytics.com/dashboard/gnosis-safe_2"
+            url="https://gnosis-safe.io/security"
```

Comment:
As per the issue, this page should link to https://dune.com/safe/ethereum, not to itself.